js/src/carousel.js

```diff
@@ -240,7 +245,9 @@ class Carousel extends BaseComponent {
       return
     }
 
-    this._slide(direction > 0 ? DIRECTION_RIGHT : DIRECTION_LEFT)
+    if (!this._isSliding) {
```

Comment:
Better to do one check for all in start of `isSliding` method 😉

Reply:
That's what I said..